packages/databricks-vscode/resources/whats-new/1.2.md

```diff
@@ -0,0 +1,48 @@
+# <!-- DATABRICKS LOGO --> Databricks Extension v1.2
+
+## What's new?
+
+-   [📗 Advanced notebook support with Databricks Connect V2](#dbconnect-notebook)
+
+## <a id="dbconnect-notebook"></a> Advanced notebook support with Databricks Connect V2
+
+You could already run notebooks locally using Databricks Connect V2. But we are now introducing advanced notebook support which enables you to run notebooks locally with the similar experience as running them on a Databricks cluster. The aim of this update is to bring most of the convenience of running notebooks on a Databricks cluster to your local environment!
+
+This includes:
+
+-   [Magic commands such as `%run` and `%sql` etc](#magic-commands)
+-   [Preconfigured globals such as spark and dbutils](#preconf-globals)
+-   [Interactive `dbutils.widgets`](#widgets)
+-   [Support for Databricks Notebooks](#dbnb)
+
+### <a id="magic-command"></a>Magic commands
+
+You can now use Databricks magic commands locally from your notebooks.
+
+See [here]() for a list of supported magic commands.
+
+![magic_sql](./1.2/magic_sql.gif)
+
+### <a id="preconf-globals"></a>Preconfigured globals
+
+Just like on a Databricks cluster, we configure and provide you with some globals such as `spark` and `dbutils` when you run notebooks locally.
+
+See [here]() for a list of supported globals.
+
+![preconf_globals](./1.2/preconf_globals.gif)
+
+### <a id="widgets"></a>Interactive `dbutils.widgets`
+
+You can now use `dbutils.widgets` to create interactive widgets in your local notebooks.
+
+See [here]() for a list of requirements.
+
+![widgets](./1.2/widgets.gif)
+
+### <a id="dbnb"></a>Support for Databricks Notebooks
+
+You can use Databricks Notebooks locally, using [VS Code Interactive Windows](https://code.visualstudio.com/docs/python/jupyter-support-py).
+
+See [here]() for more details.
```

Comment:
```suggestion
For more details, see [link-text](url).
```

Reply:
"For more details" about what? Databricks notebooks? VS Code Interactive Windows? This feature?